packages/kubernetes-1.20/kubernetes-1.20.spec

```diff
@@ -25,6 +25,7 @@ Source7: kubelet-bootstrap-kubeconfig
 Source8: kubernetes-tmpfiles.conf
 Source9: kubelet-sysctl.conf
 Source10: prepare-var-lib-kubelet.service
+Source11: credential-provider-config.yaml
```

Comment:
None of the other templated files included in Bottlerocket use the extension of the language used to write them, instead a `-` is used and no extension is provided.